defog/__init__.py

```diff
@@ -840,7 +847,7 @@ def get_query(
                 "question": question,
                 "api_key": self.api_key,
                 "previous_context": previous_context,
-                "db_type": self.db_type,
+                "db_type": self.db_type if self.db_type != "databricks" else "postgres",
```

Comment:
shall we put a note to update this section once we've updated our backend api to allow db_type=databricks?

Reply:
great point, just added on linear!